sdk/communication/Azure.Communication.NetworkTraversal/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Release History
+
+## 1.0.0-beta.1 (2021-05-25)
+
+### Added
+- Added CommunicationRelayClient in preview.
+- Added CommunicationRelayClient.GetRelayConfiguration in preview.
+- Added CommunicationRelayClient.GetRelayConfigurationAsync in preview.
+
+<!-- LINKS -->
```

Comment:
We don't need a LINKS section if there are no links to add

Reply:
Let if as we now have a [readme] linked